(Chore) Increase test coverage

fernandomg opened the pull request:
This PR is a WiP for [#165854539](https://www.pivotaltracker.com/story/show/165854539), by:
- cleaning up coverage report
- fixing overall tests and travis setup

This is the first PR of a series of progressive PRs dedicated to increase test coverage.

All branches will start with `165854539-increase-test-coverage` as a way of organization.

After mergin this one, next one will be `165854539-increase-test-coverage-goodWallet`

dminones commented on .travis.yml:
```diff
@@ -7,7 +7,7 @@ env:
     - REACT_ENV=development
     - REACT_APP_LOG_LEVEL=debug
-    - REACT_APP_NETWORK_ID=4447
+#    - REACT_APP_NETWORK_ID=4447
```
where are we getting this value if this commented out?

fernandomg replied:
From this default value:

```javascript
"networkId": process.env.REACT_APP_NETWORK_ID || 42,
```

https://github.com/GoodDollar/GoodDAPP/blob/165854539-increase-test-coverage/src/config/config.js#L6

fernandomg commented:
> It's great to be able to test blockchain related features.
> Also is great rename this kind of files so we don't have false missing tests but shouldn't we use mobile instead android in this cases?
> `src/components/dashboard/ReceiveByQR.js → ...mponents/dashboard/ReceiveByQR.android.js`

Changed, now it's `native` as we discussed. And also updated jest config files to support it.

Thanks!
